Add some information about tags in the build instructions (#386)
BeamMP/BeamMP-Server: README.md

## About Building from Source

> We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See https://github.com/BeamMP/BeamMP-Server/tags for possible versions/tags, as well as https://github.com/BeamMP/BeamMP-Server/releases to check which version is marked as a release/prerelease. We recommend to use the latest release https://github.com/BeamMP/BeamMP-Server/releases/latest

Reviewer: Could the links here be put in [text](url) format please?
For example:
```suggestion
We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See https://github.com/BeamMP/BeamMP-Server/tags for possible versions/tags, as well as https://github.com/BeamMP/BeamMP-Server/releases to check which version is marked as a release/prerelease. We recommend to use the [latest release](https://github.com/BeamMP/BeamMP-Server/releases/latest)
```

> We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See [the tags](https://github.com/BeamMP/BeamMP-Server/tags) for possible versions/tags, as well as [the releases](https://github.com/BeamMP/BeamMP-Server/releases) to check which version is marked as a release/prerelease. We recommend to use the [latest release](https://github.com/BeamMP/BeamMP-Server/releases/latest)

Reviewer: ```suggestion
We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See [the tags](https://github.com/BeamMP/BeamMP-Server/tags) for possible versions/tags, as well as [the releases](https://github.com/BeamMP/BeamMP-Server/releases) to check which version is marked as a release/prerelease. We recommend using the [latest release](https://github.com/BeamMP/BeamMP-Server/releases/latest).
```

You can build on **Windows, Linux** or other platforms by following these steps:

1. Check out the repository with git: `git clone --recursive https://github.com/BeamMP/BeamMP-Server`.

> 2. Specify the server version to build by cheking out a tag: `git checkout tags/v3.4.1` - See https://github.com/BeamMP/BeamMP-Server/tags for possible versions/tags

Reviewer: ```suggestion
2. Specify the server version to build by checking out a tag: `git checkout tags/v3.4.1` - [Possible versions/tags](https://github.com/BeamMP/BeamMP-Server/tags)
```
small spelling bit and tidied up the link

Author: Nice
